src/models/users.model.js

```diff
@@ -2,17 +2,22 @@
 //
 // See http://mongoosejs.com/docs/models.html
 // for more of what you can do here.
-module.exports = function (app) {
-  const mongooseClient = app.get('mongooseClient');
-  const users = new mongooseClient.Schema({
-  
-    email: {type: String, unique: true, lowercase: true},
-    password: { type: String },
-  
-  
-  }, {
-    timestamps: true
-  });
+module.exports = function(app) {
+  const mongooseClient = app.get("mongooseClient");
+  const users = new mongooseClient.Schema(
+    {
+      email: { type: String, unique: true, lowercase: true, required: true },
```

Comment:
It's not technically correct to enforce case sensitivity on the first part of an email:
https://stackoverflow.com/questions/9807909/are-email-addresses-case-sensitive

Domains are case insensitive, but the host part is not.  Granted, it will *likely* never cause an issue, but it can.

Reply:
Note that the `lowercase: true` was the default, a separate issue can be opened if we wish to change this, as it is unrelated to this individual pull request.

Reply:
When there is a valid case on either side, I would default to "leave it as is" however, if it all blows up later the other person can be smug.